lib/logitech_receiver/settings.py

```diff
@@ -211,6 +211,33 @@ def write(self, device, data_bytes):
 		assert self.feature is not None
 		return device.feature_request(self.feature, self.write_fnid, data_bytes)
 
+class NewFeatureRW(object):
+
+	__slots__ = ('feature', 'read_fnid', 'write_fnid', 'prefix', 'key')
+
+	kind = _NamedInt(0x02, 'feature')
+	default_read_fnid = 0x10
+	default_write_fnid = 0x20
+
```

Comment:
I worry about what fnid 0x00 is.  Again, documentation would be very useful.

@FFY00 Can we get documentation for this feature?	GESTURE_2=0x6501    Also GESTURE as well, if possible.

Reply:
I see that some other features have 0x10 for read and 0x20 for write so I'm not so worried any more, but it still would be nice to have documentation.
Is there someone at Logitech that can be asked?